From 17b4c579f3b4287b34db724fcbb81c592897d3c6 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Wed, 29 Apr 2015 14:15:18 +0800
Subject: [PATCH] MLK-10767-4 ARM: dts: imx6ul: remove power-domains in lcdif
 node

The power-domains property now is not supported properly
and will cause lcdif driver hang. So remove it.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 575c336..d677af5 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -832,7 +832,6 @@
 					 <&clks IMX6UL_CLK_LCDIF_APB>,
 					 <&clks IMX6UL_CLK_DUMMY>;
 				clock-names = "pix", "axi", "disp_axi";
-				power-domains = <&gpc 2>;
 				status = "disabled";
 			};
 
-- 
1.8.0

